docs/nodes/override-status.md

```diff
@@ -0,0 +1,7 @@
+# Override state
+
+The state can be set to ```ON```, ```OFF``` or ```TOGGLE```. Becarefull with ```TOGGLE``` because when not all Lamps in your flow are not in the same state, the ones that are on turn off and vice versa. 
```

Comment:
Does: "not all Lamps in your flow are ~~not~~ in the same state" make more sense?

Reply:
Totally agree